sections/futures/ProfitCalculator/ProfitCalculator.tsx

```diff
@@ -0,0 +1,251 @@
+import { useEffect, useState } from 'react';
+import { ethers } from 'ethers';
+import { BigNumber } from '@ethersproject/bignumber';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import PnLs from './PnLs';
+import ProfitDetails from './ProfitDetails';
+import BaseModal from 'components/BaseModal';
+import LabelWithInput from './LabelWithInput';
+import PositionButtons from '../../../sections/futures/PositionButtons';
+import { PositionSide } from '../types';
+
+const scalar = 100;
```

Comment:
I don't believe this constant is necessary if we handle the math functions below in a more elegant way.